plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,14 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getTasks() {
```

Comment:
These structs by design are not great in general. If we had a `getKey` method, we would likely either not need the struct at all since we could just access the key on the object directly, or we'd have a long switch statement checking the key for lookup. The reason why we have `getXXXX` is to provide a custom getter. Of course, now we can use getters much more elegantly but the structs are legacy. For consistency and discoverability of available methods on the object, let's keep what you have here.